src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -1118,6 +1127,8 @@ private void RenderHighlightedOrganelle(int q, int r, int rotation)
         var toBePlacedOrganelle = SimulationParameters.Instance.GetOrganelleType(
             ActiveActionName);
 
+        OrganelleCost = toBePlacedOrganelle.MPCost;
```

Comment:
This gets called a bunch of duplicate times.

I'd prefer if you put code like this in `UpdateEditor`

```
if (ActiveActionName != null && ShowHover)
{
OrganelleCost = ...
} else {
OrganelleCost = 0;
}
```